src/herder/TxQueueLimiter.cpp

```diff
@@ -110,6 +111,16 @@ TxQueueLimiter::canAddTx(TransactionFrameBasePtr const& newTx,
         newOps -= oldTx->getNumOperations();
     }
     newOps += newTx->getNumOperations();
+    if (mMinFeeNeeded.second != 0)
```

Comment:
As I commented earlier, the order here is still really weird (and the need for whitespace between different conceptual tasks remains). When I read this code, I am forced to ask myself "why did I compute the number of operations, then switch and evaluate the min fee, then switch back and actually act on the number of operations?" But you don't have to do that, so you shouldn't. Please rearrange so that the function 
1. Checks the min fee
2. Then computes `newOps` and checks if it will fit
3. Then does everything else

It'll be easier to understand for future maintainers.

Reply:
yeah agree, this was the result of a convoluted rebase dance. I'll leave this as a separate commit